### ISSUE TITLE
Audit the yellowlist for entries that can be removed

### ISSUE THREAD

**Issue by cowlicks** [EFForg/privacybadger#1593]
If a domain has been added to a mdfp group, we might be able to remove it from the yellowlist.

**Comment by ghostwords** [comment]
Removing domains from the yellowlist is inadvisable as long as pre-2017.7.24 releases are still around (https://github.com/EFForg/privacybadger/issues/1474#issuecomment-314803154).

**Comment by cowlicks** [comment]
When we upgrade the yellowlist format (to json, or a signed thing) we should make sure to include a safe way to propagate changes to older versions. While still being able to remove things.

**Comment by redranamber** [comment]
Related to this, I noticed zergnet.com and its subdomains are on the yellowlist and was trying to find a reason why. I can't see any reason why they can't be blocked and checking the logs on github didn't give any info either. So here are probably some other things other than just mdfps that can be removed too..

**Comment by ghostwords** [comment]
Yeah, `zergnet.com` is in the original "example whitelist" from 2013 contributed by @jonathanmayer. I don't see documentation around those domains.

**Comment by ghostwords** [comment]
Yandex domains probably belong in the MDFP list, not the yellowlist (at least not all `yandex.com` subdomains ...): #649.

**Comment by ghostwords** [comment]
Might be able to remove `turner.com` after most users update to 7e18dedbabeb0ea47d5db866ad16e8f52050bab7

**Comment by ghostwords** [comment]
`www.facebook.com` was removed in 39d2c1c892b27b8c0df56eaa973300a5df907d3e (see also: #2279)
